pkgs/development/interpreters/php/default.nix

```
@@ -113,8 +114,9 @@ let
       configureFlags = [
         "--with-config-file-scan-dir=/etc/php.d"
       ]
-      ++ optional (versionOlder version "7.3") "--with-pcre-regex=${pcre.dev} PCRE_LIBDIR=${pcre}"
-      ++ optional (versionAtLeast version "7.3") "--with-pcre-regex=${pcre2.dev} PCRE_LIBDIR=${pcre2}"
+      ++ optional ((versionOlder version "7.3") && (!versionAtLeast version "7.4")) "--with-pcre-regex=${pcre.dev} PCRE_LIBDIR=${pcre}"
```

`v < 7.3 && !(v ≥ 7.4) = v < 7.3 && v < 7.4 = v < 7.3`

```suggestion
      ++ optional (versionOlder version "7.3") "--with-pcre-regex=${pcre.dev} PCRE_LIBDIR=${pcre}"
```

---

In version 7.3 uses the parameter `"--with-pcre-regex"`, and in the version 7.4 `"--with-external-pcre"`

---

Any reason why not the bundled one?

---

See https://fedoraproject.org/wiki/Bundled_Libraries

---

Well, the versions covered by `< 7.3` are a subset of those covered by `!(v ≥ 7.4)`, so the second conjunct is not really needed.

---

fixed, thanks!